scripts/vmpublish.rb

```diff
@@ -0,0 +1,80 @@
+require 'logger'
+require 'pathname'
+require 'trollop'
+require 'json'
+
+require_relative 'config'
+
+
```

Comment:
If we keep this separate, that's fine, but this should be in a `Publish` module.